index.bs

```diff
@@ -2013,6 +2013,9 @@ parameter to the {{CredentialsContainer/create()}} or {{CredentialsContainer/get
     The <dfn>type</dfn> member contains the type of the [=public key credential=] the caller is referring to.
 
     The <dfn>id</dfn> member contains the [=credential ID=] of the [=public key credential=] that the caller is referring to.
```

Comment:
while we're at it, we might delete the "that" in the above sentence :)

Reply:
Done!